Frameworks/CoreGraphics/CGColor.mm

```diff
@@ -62,18 +68,20 @@ CGColorRef CGColorRetain(CGColorRef color) {
 }
 
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes only supports RGB
 */
 CGColorRef CGColorCreate(CGColorSpaceRef colorSpace, const CGFloat* components) {
-    CGColorRef ret =
-        (CGColorRef)[[__LazyUIColor colorWithRed:components[0] green:components[1] blue:components[2] alpha:components[3]] retain];
-
+    RETURN_NULL_IF(!colorSpace);
+    RETURN_NULL_IF(!components);
+    CGColorRef ret = static_cast<CGColorRef>(
```

Comment:
hmm i don't want to do this, as it's only supporting RGB, not even grayscale.
This is not the main purpose of the CR. The main purpose is to move it to Add CGColorGetColorSpace only.
These will be implemented properly in the next iteration.

---
In reply to: [106506478](https://github.com/Microsoft/WinObjC/pull/2244#discussion_r106506478) [](ancestors = 106506478)